horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/UtilTest.java

```diff
@@ -61,22 +115,99 @@ public void toInstant_nulls() throws UnsupportedEncodingException {
             assertFalse(handler.getLog().contains("DateTimeParseException"),handler.getLog());
         }
     }
+
     @Test
     public void toInstant_valid() throws IOException {
         //already an instant
-        assertNotNull(Util.toInstant(Instant.now()),"failed to handle instant input");
+        assertNotNull(Util.toInstant(Instant.now()), "failed to handle instant input");
         //number
-        assertNotNull(Util.toInstant(System.currentTimeMillis()),"failed to parse current millis");
+        assertNotNull(Util.toInstant(System.currentTimeMillis()), "failed to parse current millis");
         //strings
-        assertNotNull(Util.toInstant(""+System.currentTimeMillis()),"failed to parse millis as string");
-        assertNotNull(Util.toInstant("2020-01-01"),"failed to parse yyyy-mm-dd");
-        assertNotNull(Util.toInstant("2020-01-01T01:01:01"),"failed to parse YYYY-MM-DDTHH:mm:ss");
-        assertNotNull(Util.toInstant("2020-01-01T01:01:01Z"),"failed to parse YYYY-MM-DDThh:mm:ssZ");
-        assertNotNull(Util.toInstant("2020-01-01T01:01:01+00:00"),"failed to parse YYYY-MM-DDThh:mm:ss[+-]zz:zz");
-        assertNotNull(Util.toInstant("2024-03-11T22:16:24.302655-04:00"),"failed to parse ISO with microseconds and zzzz");
+        assertNotNull(Util.toInstant("" + System.currentTimeMillis()), "failed to parse millis as string");
+        assertNotNull(Util.toInstant("2020-01-01"), "failed to parse yyyy-mm-dd");
+        assertNotNull(Util.toInstant("2020-01-01T01:01:01"), "failed to parse YYYY-MM-DDTHH:mm:ss");
+        assertNotNull(Util.toInstant("2020-01-01T01:01:01Z"), "failed to parse YYYY-MM-DDThh:mm:ssZ");
+        assertNotNull(Util.toInstant("2020-01-01T01:01:01+00:00"), "failed to parse YYYY-MM-DDThh:mm:ss[+-]zz:zz");
+        assertNotNull(Util.toInstant("2024-03-11T22:16:24.302655-04:00"), "failed to parse ISO with microseconds and zzzz");
         //json
-        assertNotNull(Util.toInstant(new TextNode("2020-01-01")),"failed to parse json text YYYY-MM-DD");
-        assertNotNull(Util.toInstant(new LongNode(System.currentTimeMillis())),"failed to parse current millis as json node");
+        assertNotNull(Util.toInstant(new TextNode("2020-01-01")), "failed to parse json text YYYY-MM-DD");
+        assertNotNull(Util.toInstant(new LongNode(System.currentTimeMillis())), "failed to parse current millis as json node");
+    }
+    @org.junit.jupiter.api.Test
+    public void evaluateOnceJsonKeyAccess() throws JsonProcessingException {
+        Object rtrn = Util.evaluateOnce(
+            """
+                    (input)=>{
+                        return input.foo;
+                    }
+                    """,
+                new ObjectMapper().readTree("{\"foo\":\"bar\"}"), //no input needed
```

Comment:
I think just a leftover
```suggestion
                new ObjectMapper().readTree("{\"foo\":\"bar\"}"),
```